bin/plotting/pycbc_plot_dq_flag_likelihood

```diff
@@ -37,24 +41,24 @@ color = pycbc.results.ifo_color(ifo)
 
 fig = pyplot.figure(0)
 ax = fig.add_subplot(111)
-ax.bar(x, height=logrates, label=f'{ifo} DQ Flag Trigger Rates',
-       color=color)
-ax.set_xlabel('Template Bin Number')
-ax.set_ylabel('Data Quality Log Likelihood')
-ax.set_xticks(x)
-ax.legend(loc='upper left', markerscale=5)
-ax.set_ylim(0, ymax)
-ax.grid()
+ax2 = ax.twinx()
+
+ax2.bar(x, height=logrates, label=ifo, color=color)
+ax2.set_ylabel('DQ Log Likelihood')
```

Comment:
Actually, this is a log likelihood _penalty_, right? I.e. it is a number that gets subtracted from the ranking stat? 
If so, I think it would be preferable to put that in the caption

Reply:
Ok I have added the word 'Penalty' to the end of this label